discord/guild.py

```diff
@@ -3379,22 +3379,22 @@ async def query_members(
         ----------
         query: Optional[:class:`str`]
             The string that the username's start with.
-        limit: :class:`int`
-            The maximum number of members to send back. This must be
-            a number between 5 and 100.
-        presences: :class:`bool`
+        user_ids: Optional[List[:class:`int`]]
+            List of user IDs to search for. If the user ID is not in the guild then it won't be returned.
+
+            .. versionadded:: 1.4
+        limit: Optional[:class:`int`]
+            The maximum number of members to send back. If no query is passed, pass ``None`` to return all members.
+            If you specify a ``query`` or ``user_ids``, this must be between 1 and 100. Defaults to 5.
+        presences: Optional[:class:`bool`]
             Whether to request for presences to be provided. This defaults
             to ``False``.
 
             .. versionadded:: 1.6
 
-        cache: :class:`bool`
+        cache: Optional[:class:`bool`]
```

Comment:
This is not optional. It defaults to `True` which is a bool.
```suggestion
        cache: :class:`bool`
```